businesscentral-monitoring/image.yaml

```diff
@@ -137,6 +138,7 @@ modules:
           - name: jboss-kie-workbench
 packages:
       content_sets_file: content_sets.yml
+      manager: microdnf
       install:
           - hostname
```

Comment:
remove the hostname package from the image.yaml as it is being installed by the common.dependencies module.